src/components/Molecules/EmailSignUp/EmailSignUp.js

```diff
@@ -24,8 +24,16 @@ const EmailSignUp = ({
   ...rest
 }) => {
   const [value, setValue] = useState('');
+
+  const handleSubmit = e => {
+    // Just stopping weird things happening for now when the user submits with the enter key
+    // rather than the Submit button
+    e.preventDefault();
+    console.log('');
```

Comment:
May return something here instead of the console log

Reply:
Oh yeah whoops, was editing that and didn't finish haha